src/clocksi_static_tx_coord_sup.erl

```diff
@@ -28,7 +28,7 @@
 
 -export([init/1]).
 
--define(NUM, 100).
+-define(NUM, 1).
```

Comment:
Do you know why?  On the evaluation branch with my changes, the throughput when from under 4000 to 5000 with my changes.  I'm not going to merge this code until we know *why* that change is making something happen, given it's a pool of supervisors that it randomly balances on.  I want to know why this change makes a difference.